src/components/TeamProfileCards/index.js

```diff
@@ -89,20 +89,21 @@ export function ActiveTeamRow() {
           Ari is the creator and core maintainer of Start Your Lab. He is also an incoming Ph.D. student in Social Psychology and a Centennial Fellow at Princeton University (starting Fall 2021). Ari's research focuses on the application of artificial intelligence to facilitate collective problem-solving and collaborative processes on a global scale. With a breadth of experiences across physics, mathematics, finance, and software, he hopes to bring modern tools to the forefront of the social sciences, with a nod toward increased interdisciplinary work between fields and industries.
         </Translate>
       </TeamProfileCardCol>
-      {/*<TeamProfileCardCol
-        name="Peter Sokol-Hessner"
+      <TeamProfileCardCol
+        name="Peter Sokol-Hessner (he/him/his)"
```

Comment:
This reminds me that we might even want a separate key-value field _just_ for pronouns, which from a responsive web perspective will make the most sense (allows us to style them separately and on different lines if needed for mobile width screens).

Reply:
I'll need to implement it separately, so no need to do so here prior to review approval.

Reply:
Yes, agreed, had no idea how to do that so did the lame version.

Reply:
Made a resolution to this with an additional commit, and also specified a new `avatar__pronouns` class in our CSS customization